_posts/2020-08-12-DDoS-criminal-hackers.md

```diff
@@ -0,0 +1,68 @@
+
+The documents, first obtained and published by the transparency non-profit
+Distributed Denial of Secrets (DDoS), provide an intimate look into police
+agencies and fusion centers across the country. Established after 9/11 to
+disrupt terrorism plots, fusion centers have instead been challenged for massive
+privacy and civil liberties violations. DDoS obtained over 270 gigabytes of data from
+fusion centers and police departments and published the release in what it
+billed the #BlueLeaks release.
+
+Journalists and researchers immediately seized on the Blue Leaks release to
+reveal that police were obsessed with left-wing movements and ignoring
+far-right [domestic extremism](https://theintercept.com/2020/07/15/george-floyd-protests-police-far-right-antifa/).
+Blue Leaks further revealed that law enforcement inflated threats after the
+police murder of George Floyd in Minneapolis with some police stating, without evidence, that "Antifa"
+would carry out [car bombings](https://theintercept.com/2020/06/26/blueleaks-minneapolis-police-protest-fears/).
+The shoddy intelligence of fusion centers has been [well documented](https://www.hsgac.senate.gov/imo/media/doc/10-3-2012%20PSI%20STAFF%20REPORT%20re%20FUSION%20CENTERS.2.pdf)
+for over fifteen years.  
+
+Nonetheless, authorities moved swiftly to crack down on DDoS's right to publish, first by [seizing its server](https://www.infosecurity-magazine.com/news/german-police-seize-blueleaks/)
+in the middle of the night presumably on behalf of the FBI. Homeland Security
+Investigations (HSI), one of the largest agencies within DHS, visited
+unaffiliated individuals hosting an archives of the Blue Leaks and even [offered them money](https://www.vice.com/en_us/article/xg84ww/ice-questions-admin-of-the-eye-archive-site-hosted-blue-leaks) to become informants. Just this week, documents obtained by Lucy Parsons Labs [confirm](https://cdn.muckrock.com/foia_files/2020/08/10/CCSO_00000003.pdf) DHS considers DDoS "criminal hackers" and not journalists.
+
+
+The government's aggressive push to name DDoS as "criminal hackers" is a
+dangerously unprecedented territory that criminalized everyone's right to
+publish online under the First Amendment. It is not illegal to possess and
+publish information obtained by a third party, even if it was obtained
+illegally, under the First Amendment as long as an individual does not
+participate in the illegal activity. This extends even to the publishing of
+classified information, as the Supreme Court
+[upheld](https://en.wikipedia.org/wiki/New_York_Times_Co._v._United_States) in
+the infamous case of the Pentagon Papers. (Almost all of the data published in
+the Blue Leaks was
+[unclassified](https://www.dla.mil/Portals/104/Documents/InformationOperations/LogisticsInformationServices/Customer%20Outreach/Subscriptions/J6_ForOfficialUseOnly_160113.pdf).)
+
+The confirmation of an investigation by DHS's Office of Intelligence and
```

Comment:
Is it technically correct to say there's an "investigation," when what we're looking at is an intelligence bulletin?

Reply:
You are correct that the intelligence bulletin alone is not enough to confirm an official investigation.  The evidence here is the bulletin, coupled with HSI claiming DDoS did a "hack and leak" (with little evidence), the fact that the server was seized out of a German data center through an MLAT _and_ because HSI visited others asking them to become informants. So I feel comfortable saying investigation here.